### PR TITLE
add print statment

### PR DESCRIPTION
**Pull request by jrcallingham**

### CODE REVIEW

File: get_fluffy.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Fluffy Happiness: Test code to grab pictures of cute animals from the Internet
 # Usage: >> python get_fluffy.py [options]
 # V.A. Moss (vmoss.astro@gmail.com)
-__author__ = "V.A. Moss"
+__author__ = "V.A. Moss smells"
```
Comment (cosmicpudding) [inline get_fluffy.py]: This is a sub-optimal statement

File: get_fluffy.py [hunk 2]
```diff
@@ -38,6 +38,7 @@
 for line in read_response:
 
 	line = line.decode('utf-8')
+	print(line)
```
Comment (cosmicpudding) [inline get_fluffy.py:41]: I can see why you might want to print this line

File: get_fluffy.py (hunk 1 above)
Comment (jrcallingham) [reply get_fluffy.py]: But it is an objectively true statement.

Comment (cosmicpudding) [reply get_fluffy.py]: Maybe we should qualify what we mean so that it is not vague. I suggest "V.A. Moss smells nice"

Comment (jrcallingham) [reply get_fluffy.py]: I find that an acceptable compromise. I accept and ill add that next merge

Comment (cosmicpudding) [reply get_fluffy.py]: Tally ho, ol' chap.

**Pull request merged**